projects/uitdatabank/docs/search-api/advanced-queries.md

```diff
@@ -0,0 +1,1067 @@
+```
+GET /places/?q=bookingAvailability:unavailable
+```
+
+TO DO: link to booking availability guide when that guide is finished.
+
+### calendarType
+
+Every event and place in UiTdatabank has one of the following four calendarTypes:
+
+* `Single`: the event occurs on a single date, indicated by a single startDate and endDate.
+* `Multiple`: the event occurs on multiple dates, and has multiple subEvent entries with each a different startDate and endDate.
+* `Periodic`: the event or place runs for a specific period as indicated by its startDate and endDate, and can optionally have openingHours.
+* `Permanent`: the event or place is permanent and has no startDate or endDate, but it can optionally have openingHours.
+
+With the `calendarType` field you can look for results that match a certain calendarType.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers`
+
+**Possible values**
+
+`single` `multiple` `periodic` `permanent`
+
+**Example**
+
+Retrieve all permanent events and places:
+
+```
+GET /offers/?q=calendarType:permanent
+```
+
+### completedLanguages
+
+All documents created in UiTdatabank are available in Dutch (NL). Besides Dutch, content editors can use UiTdatabank to translate the name and description of their documents in French (FR), German (DE) and English (EN). The API supports translations in every [two-letter language ISO 639-1 code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).
+
+You can use the `completedLanguages` field to only look for documents that are fully translated to a specific language.
+
+> Note that if an optional field is left empty in all languages, a translation can still be considered to be complete if all other fields that have a value are translated.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+Allowed values for both are any of the two-letter language ISO 639-1 codes, but the most used languages are: `nl` `fr` `de` `en`
+
+**Example**
+
+Retrieve all organizers that are fully translated in English:
+
+```
+GET /organizers/?q=completedLanguages:en
+```
+
+### created
+
+UiTdatabank keeps track of the creation dates of each document. This metadata for the creation date can be queried with the `created` field.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+Any ISO-8601 datetime, e.g. `2023-04-01T12:08:01+01:00`
+
+<!-- theme: info -->
+
+> Note that the `+` sign must be encoded for URLs (as `%2B`). Otherwise it will be interpreted as whitespace and the given date time will be considered invalid.
+
+**Example**
+
+Get all events and places that are created since the start of 2022:
+
+```
+GET /offers/?q=created:[2022-01-01T00\:00\:00%2B01\:00 TO *]
+```
+
+### creator
+
+Use the `creator` field to search for documents created by a specific creator.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+Any uuid of a creator.
```

Comment:
(Copied from older comment on previous filename)

It is only a uuid in the case of UiTID v1 creators. In the case of UiTID v2 users (that did not exist in UiTID v1), it can be a lot of things. For example:

- `google-oauth2|117412536054327843587` for Google logins
- `facebook|10159360927408483` for Facebook logins
- `auth0|5f03150966abde00196567e5` for email + password logins

And for m2m tokens it will be something like `Nnx3vGNuVVX2KJdtzHhwnx15WlTqGCZ9@clients`

(I've used fake ids here since this is a public repo)

I wouldn't try to explain all of these different formats as we cannot control them. Instead we can explain that they can look at the `creator` field of events/places, or the `id` field when they access the `/user` endpoint with  a user token. For client tokens there is currently no good way to find the value that they creator would have for a given token, but we could add one like a `/client` endpoint

Also if we specify that this is specifically a UUID, some integrators may depend on that format in their code (for example I've seen an integrator complain that some events did not have a valid UUID as id because they typed their database column as a hexadecimal field specifically, while some events from UDB2 do not have a valid UUID and thus would not be accepted by that database field). It's better to just treat this as an opaque string that can be anything, and then it doesn't matter to integrators if it's sometimes a UUID, an email address, or one of those auth0 ids.

Reply:
I replaced `uuid` with `id` or `identifier` (according to the context / sentence)